core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,106 +1,106 @@
 <div class="oppia-static-header">
-  <h1 translate="I18N_ABOUT_PAGE_HEADING"></h1>
+  <h1 [innerHTML]="'I18N_ABOUT_PAGE_HEADING' | translate"></h1>
```

Comment:
Yeah, for some reason our custom linter finds `innerHTML` in ts not acceptable but it fine with innerHTML being in an HTML file. +cc @Hudda.

Reply:
I think @kevintab95 would be the best person to answer this question as he is the one who made this change.

Reply:
I think it's happing because we have bad pattern check for it: 
![image](https://user-images.githubusercontent.com/16653571/84416155-75ec0b00-ac31-11ea-992f-091a471021ad.png)

Will it be fine to add the i18n service file to the exclude_files list?

Reply:
I would say a new pattern is needed here. `innerHTML` is now allowed in our codebase when followed by `| translate`.

Reply:
We don't allow innerHTML in JavaScript because it can lead to insecure XSS issues. This practice will continue. However, it is fine to have it in HTML as an attribute name.